mcs/build/profiles/mobile_static.make

```diff
@@ -40,8 +40,12 @@ MOBILE_STATIC = yes
 MOBILE_PROFILE = yes
 NO_VTS_TEST = yes
 
+# Unity : disable signing
+NO_SIGN_ASSEMBLY = yes
+
 # Note need for trailing comma. If you add, keep it
 PROFILE_TEST_HARNESS_EXCLUDES = MobileNotWorking,PKITS,
 
-ALWAYS_AOT = yes
+# Unity : don't generate aot assemblies as part of the profile build
+#ALWAYS_AOT = yes
```

Comment:
I am not sure we should be modifying upstream files here vs creating our own profile. Maybe fine for now...